style.css

```diff
@@ -130,3 +130,264 @@ i:hover {
+  font-family: 'Crete Round', 'Times New Roman', sans-serif;
+  font-style: normal;
+  font-size: 36px;
+  line-height: 52px;
+  text-align: left;
+}
+
+hr {
+  margin-bottom: 20px;
+}
+
+.project-section {
+  background:
+    linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%),
+    url("images/Img\ Placeholder\ \(2\).png");
+  left: 0;
+  top: 0;
+  width: 80%;
+  height: 369px;
+  color: #fff;
+  border-radius: 0;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-end;
+  flex-direction: column;
+  gap: 20px;
+  padding: 20px;
+  margin: 20px;
+  letter-spacing: 1.2px;
+}
+
+.project-box h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 32px;
+  line-height: 44px;
+  color: #000;
+}
+
+.project-box p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.project-box {
+  background-color: #fff;
+  display: flex;
+  width: 80%;
+  gap: 10px;
+  align-items: space-around;
+  flex-direction: column;
+}
+
+.btn ul {
+  border: none;
+  outline: none;
+  color: rgba(255, 255, 255, 0.9);
+  height: 40px;
+  width: 100%;
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.btn li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 12px;
+  width: 78px;
+  height: 40px;
+  color: #000;
+  border: 1px solid #8993a4;
+}
+
+.project-section li {
+  color: #fff;
+  width: 90px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.project-section h2 {
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.btn {
+  width: 100%;
+}
+
+.see_more_btn {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 80%;
+}
+
+.see_more_btn button {
+  width: 140px;
+  height: 48px;
+  color: #fff;
+  outline: none;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  background: #f55800;
+  padding: 12px;
+}
+
+.see-btn button,
+.resume-btn button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  height: 55px;
+  width: 329px;
+  position: relative;
+  left: -20.9px;
+  top: 2px;
+  margin-bottom: -20px;
+  outline: none;
+  border: none;
+  color: #fff;
+  background: #f55800;
+}
+
+button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  transition: ease-in 1s;
+}
+
+.button:active {
+  background: #e05304;
+  background: #f4f1f0;
+  transform: translateY(8px);
+}
```

Comment:
[OPTIONAL]

- [ ] Your active styling here is not working for two reason
- First you are using a class selector `.button` along with the `:active` however there is no place in your html where you defined a `button` class so this styling is not even working at all
- Secondly you have styled two colors for the background and based on how css styling cascades from the top to the bottom the last written styling is what will be used which currently is `background: #f4f1f0;`
- To fix your code use the example below. :+1: 
```
  button:active {
  background: #e05304;
  transform: translateY(8px);
}
```
- [ ] Great job overall working on your project you have done extremely well and i am glad to approve your work :sparkles: :grin: :+1: